arch/arm/src/stm32h7/stm32_fmc.c

```diff
@@ -39,40 +39,387 @@
 
 #include <nuttx/config.h>
 
+#if defined(CONFIG_STM32H7_FMC)
+
 #include "stm32.h"
 
-#if defined(CONFIG_STM32H7_FMC)
+#include <debug.h>
+
+//#include "stm32_fmc.h"
+//#include "hardware/stm32_pinmap.h"
+//#include "stm32_gpio.h"
+//#include "stm32_rcc.h"
```

Comment:
Use of commented out code and any use of C++ comments is forbidden by the coding standard.

Reply:
Sorry, its a leftover from the merge from my other git repo, and forgot to remove them after checking that it works. Will fix that now.